qiita_db/software.py

```diff
@@ -508,6 +515,22 @@ def activate(self):
             qdb.sql_connection.TRN.add(sql, [True, self.id])
             return qdb.sql_connection.TRN.execute()
 
+    @property
+    def analysis_only(self):
+        """Returns if the command is an analysis-only command
```

Comment:
Can a command be both an analysis command and "other"? I guess I don't understand what is under the umbrella of "analysis" and what is not